Mega-form

SanaKhan1997 opened the pull request:
Contains merged information from CFIA forms i.e. Animal Record, FWR, Transfer of Care and Contingency Plan

SanaKhan1997 committed:
Mega-form

Contains merged information from CFIA forms i.e. Animal Record, FWR, Transfer of Care and Contingency Plan

holtzmak reviewed (commented):
Nice to see all the form elements together. Could you explain to me how to read the overlap flags you added? For example:

_Receiver’s Contact number in case of emergency:_ __________CP_

Is it just this line in particular that overlaps with the Contingency Plan, what about the other elements near it like:

_PID: Name:_
_Address:____________

and 

_Date of arrival (dd/mm/yyyy):____________
_Time of arrival:_______ _AM/PM_______
_Arrival: All animals arrived in good condition ? Yes/No_

?

SanaKhan1997 commented:
I've organized information according to heading given i.e.

1. Shipper's Info
2. Transporter's Info
3. Receiver's Info
4. FWR
5. Loading Info
6. Contingency Plan

If a color tag is beside a form element it is only specific to that form element and does not affect anything around. But if a tag is beside a heading(one from above), it covers every form element in that heading.

holtzmak reviewed (approved):
Thanks for clearing up my confusion, I had originally thought that was how we should read the document but wanted to be sure. Before merging this, there's a few more things I'm wondering about (that you may or may not want to do):

1. Maybe we shouldn't call it the "mega-form" in the document itself, we the team know what "mega-form" means but we should probably call it what it is: "Animal Transport Record Form with Overlap Signifiers" (or similar, simpler).
2. Should the form be in the same color format as our other documentation files in the documentation folder? e.g. the other forms have orange, our names on the documentation, a left-aligned header. Tim had mentioned that he likes attention to detail. Do you think we could do that (maybe too nitpicky)?
3. I would change the file name to match your decision against # 1

SanaKhan1997 commented:
Alright! I'm going to change the document template to the one we've been using and for the name part how does "Animal Transportation Form" sound and I can specify CFIA forms it contains in the description.

holtzmak commented:
> Alright! I'm going to change the document template to the one we've been using and for the name part how does "Animal Transportation Form" sound and I can specify CFIA forms it contains in the description.

That's perfect

SanaKhan1997 committed:
Delete MEGA-FORM.pdf

Template changed

SanaKhan1997 committed:
Add files via upload

Mega form named "Animal Transportation Form" with selected documentation template.

SanaKhan1997 commented:
Here's the updated mega-form [Animal Transportation Form](https://github.com/holtzmak/Humane-Transport/blob/SanaKhan1997-patch-1/documentation/Animal%20Transportation%20Form-(Mega%20Form).pdf)
@holtzmak

holtzmak commented:
@SanaKhan1997 The only thing worth noting is that the "Animals Loaded" Table appears cut-off in the PDF.

SanaKhan1997 committed:
Delete Animal Transportation Form-(Mega Form).pdf

SanaKhan1997 committed:
Table corrected

SanaKhan1997 committed:
Animal transportation form sequence

holtzmak commented on documentation/Megaform Sequence Diagram.svg:
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
What is this? Does this need to be under revision control?

holtzmak replied:
I see now that this is the sequence diagram drawing. Can we make it a file type other than svg? This file type is a strange one to put under revision control (plus, it's basically one super long line so we should at least format it to be readable)

SanaKhan1997 replied:
I was planning to put the image in the document template and upload it as pdf if that's fine.

holtzmak replied:
Yeah, that's what I'd expected, too

SanaKhan1997 committed:
Sequence diagram converted to pdf format

SanaKhan1997 committed:
Delete Megaform Sequence Diagram.svg

svg is file replaced with pdf format as "Sequence Diagram(user's perspective)"
